app/forms/ZeroRatedSuppliesNewJourneyForm.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms
+
+import forms.FormValidation._
+import play.api.data.Form
+import play.api.data.Forms._
+
+object ZeroRatedSuppliesNewJourneyForm {
+
+  implicit val errorCode: String = "zeroRatedSupplies.newJourney"
+  val zeroRatedSuppliesKey = "zeroRatedSupplies"
+  val regex = """^[0-9 ,]*\.?[0-9]+$""".r
```

Comment:
This regex is causing negative numbers to give the error 'Not a number' rather than an 'Out of 0 - turnover range'.
Has this been approved by business or should we make the regex `^-?[0-9 ,]*\.?[0-9]+$`? Will ask the group

Reply:
Spoke with @kannanEkambaram , this is something I shall bring up to the BAs to get their opinion on and we can raise a new ticket to correct if needed.